pepper-apis/docs/specification/src/endpoints/admin.studies.invitation-lookup.yml

```diff
@@ -27,53 +27,26 @@ post:
       content:
         application/json:
           schema:
-            type: object
-            required:
-              - invitationId
-              - createdAt
-              - voidedAt
-              - verifiedAt
-              - acceptedAt
-              - notes
-              - userGuid
-              - userHruid
-              - userLoginEmail
-            properties:
-              invitationId:
-                description: the id of the invitation
-                type: string
-              createdAt:
-                description: timestamp of when invitation was created, in ISO-8601 UTC format
-                type: string
-                example: 2020-05-13T17:58:33.887Z
-              voidedAt:
-                description: timestamp of when invitation is voided, in ISO-8601 UTC format
-                type: string
-                nullable: true
-              verifiedAt:
-                description: timestamp of when invitation is verified, in ISO-8601 UTC format
-                type: string
-                nullable: true
-              acceptedAt:
-                description: timestamp of when invitation is accepted, in ISO-8601 UTC format
-                type: string
-                nullable: true
-              notes:
-                description: the notes for the invitation
-                type: string
-                nullable: true
-              userGuid:
-                description: guid of associated user, if invitation is accepted
-                type: string
-                nullable: true
-              userHruid:
-                description: hruid of associated user, if invitation is accepted
-                type: string
-                nullable: true
-              userLoginEmail:
-                description: login email of associated user, if invitation is accepted and user has a login account
-                type: string
-                nullable: true
+            allOf:
+              - $ref: '../pepper.yml#/components/schemas/Invitation'
```

Comment:
In the spirit of reuse, we're not extending the `Invitation` object. This has side-effect of pulling in a new `invitationType` property. It's not a breaking change and it shouldn't affect current callers of this API, so it should be fine.